shotgun_api3/shotgun.py

```diff
@@ -186,6 +186,12 @@ def ensure_per_project_customization(self):
             'label': 'project parameter'
         }, True)
 
+    def ensure_support_for_additional_server_presets(self):
+        """Wrapper for ensure_support"""
+        return self._ensure_support({
+            'version': (6, 3, 11),
+            'label': 'project parameter'
```

Comment:
The string with the label says ```project parameter```.  It should be something for the presets.  Just making sure that's the bit you were planning on updating before a merge.